.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 .idea
 Cargo.lock
+grove.db
```

Comment:
I guess we moved to temporary dirs for tests, do we need to ignore `grove.db` for some reason?